shell/plugins/steve/resourceWatcher.js

```diff
@@ -40,10 +40,10 @@ export const WATCH_STATUSES = {
  * Create a unique key for a specific resource watch's params
  */
 export const keyForSubscribe = ({
-  resourceType, type, namespace, id, selector
+  resourceType, type, namespace, id, selector, mode
```

Comment:
subscribe watch keys now include `mode`, allows for concurrent mode and no mode watches